package.json

```diff
@@ -48,6 +48,7 @@
     "@vladmandic/human": "2.6.5",
     "@vladmandic/human-models": "2.5.9",
     "@xmldom/xmldom": "0.8.7",
+    "abab": "^2.0.6",
```

Comment:
Pin it to the exact version pl.

Reply:
Ah damn it. I thought I removed that caret.

Reply:
Done.